hedera-node/hedera-evm-api/src/main/java/com/hedera/services/evm/contracts/execution/HederaEvmMessageCallProcessor.java

```diff
@@ -98,4 +103,7 @@ protected void executeHederaPrecompile(
             frame.setState(EXCEPTIONAL_HALT);
         }
     }
+
+    protected abstract void executeLazyCreate(
+            final MessageFrame frame, final OperationTracer operationTracer);
```

Comment:
Not sure whether we should have this `executeLazyCreate` method in the EVM module or just directly have all the lazy create logic entirely on the consensus node's `HederaMessageCallProcessor`. 

Will the mirror node need its own version of lazy creates for transaction replays or for something else? @IvanKavaldzhiev @shemnon

Reply:
I believe we can have a placeholder method like the one you introduced but not make it abstract, since we don't want the class to be abstract. It can have an empty body for now and the `MessageCallProcessor` would still override it. 

I think the mirror-node would need light-weight version of this method but we can implement it at a later point, when we introduce `eth_estimateGas`.